### PR TITLE
Playlist: fix odd length string playlist publish error

### PR DESCRIPTION
**Pull request by saltrafael**

### CODE REVIEW

**Comment by tzarebczan** [comment]
See slack, still broken

**Comment by tzarebczan** [comment]
@saltrafael can you get these 2 prs merged or whatever the best way for kp to review is?

File: ui/redux/selectors/collections.js [hunk 1]
```diff
@@ -508,6 +517,9 @@ export const selectFeaturedChannelsByChannelId = createSelector(
     });
 
     Object.values(publicLists).forEach((col) => {
+      // $FlowFixMe
+      if (col.key) return;
+
       // $FlowIgnore
       const { type, id } = col;
```
Comment (infinite-persistence) [inline ui/redux/selectors/collections.js:524]: `forEach` cannot be broken or returned early.  It's undefined behavior.

**Review by infinite-persistence — COMMENTED** [review]
If you like, I can look at the rest tomorrow, but would need some description of the issue to help test.

File: ui/page/collection/index.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { makeSelectClaimForClaimId } from 'redux/selectors/claims';
```
Comment (infinite-persistence) [inline ui/page/collection/index.js:5]: Not sure if legacy or not, but I tried to copy one of my public playlist and then tried to publish ... it says list is empty but the wallet shows items are correctly populated.

image

Comment (infinite-persistence) [reply ui/page/collection/index.js:5]: Not legacy -- changed to master and the publish form looks correct (allows me to publish)

**Pull request closed without merging**